tutorials/Flop_Count_Example.ipynb

```diff
@@ -0,0 +1,327 @@
+    "\n",
+    "len(unsupported_ops) == 0 # Verifying no unsupported operations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3ce26204",
+   "metadata": {},
+   "source": [
+    "However, if you need to add support for more ops or override the counter's default implementation, read on!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52f36115",
+   "metadata": {},
+   "source": [
+    "## Advanced Usage\n",
+    "\n",
+    "### How the Counter Works\n",
+    "\n",
+    "The flop counter in `neuralcompression` makes heavy use of the counting utilities in [fvcore](https://github.com/facebookresearch/fvcore). Counting a model's flops is a two-step process:\n",
+    "\n",
+    "1. Using PyTorch's [TorchScript](https://pytorch.org/docs/stable/jit.html) capabilities, the model is first JIT-traced into a computational graph. Each node in the graph corresponds to an ATen (linear algebra) operation, like matrix multiplications, convolutions, and elementwise operations like additions/subtractions.\n",
+    "\n",
+    "\n",
+    "2. Every node in the traced graph is iterated over, and if a node is associated with a registered flop-counting function, that function is invoked and the counted flops are added to the model's total.\n",
+    "\n",
+    "\n",
+    "In this second step, the counter's registered flop functions are stored as a dictionary mapping operator names (e.g. `aten::add`, `aten::matmul`) to counter functions. These functions have a signature of:\n",
+    "\n",
+    "```\n",
+    "def my_counter_function(inputs: List[torch._C.Value], outputs: List[torch._C.Value]) -> float\n",
+    "```\n",
+    "\n",
+    "where objects of type `torch._C.Value` represent the symbolic inputs and outputs for the node in the graph (i.e. they're not actual concrete `Tensor`s). Check out the `fvcore` codebase for [examples of how to write counter functions](https://github.com/facebookresearch/fvcore/blob/master/fvcore/nn/jit_handles.py).\n",
+    "\n",
+    "### Customizing the Counter\n",
+    "\n",
+    "To register additional counter functions or override the counter's default implementation for specific operations, use the `counter_overrides` argument, which takes the form of a dictionary mapping operator names to the corresponding counter functions you wish to use. As an example, consider the following simple module:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "aa3b8c44",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class MyModule(nn.Module):\n",
+    "    def forward(self,x,y,z):\n",
+    "        return x + y * z"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fdb5c1db",
+   "metadata": {},
+   "source": [
+    "By default, the elementwise addition or multiplication of two tensors of shape `N x M` will contribute `N x M` flops to the total computational complexity of a model (since each scalar addition/multiplication is counted as 1 flop):"
```

Comment:
So actually, in AI when we report FLOPs, we actually mean Multiply-Accumulate operations (https://en.wikipedia.org/wiki/Multiply%E2%80%93accumulate_operation), which is roughly half the "conventional" FLOPs. So the statement should be "addition and multiplication". Maybe you can mention multiply acc in the tutorial too!

See the ResNeXt paper as an example (https://arxiv.org/pdf/1611.05431.pdf) - 
> in terms of FLOPs (floating-point operations, in # of multiply-adds)